mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -293,8 +296,10 @@ public void testJoinTablesSupportedTypes() throws IOException {
   public void testSelectDistinctFromTable() throws IOException {
     for (int i = 0; i < SUPPORTED_TYPES.size(); i++) {
       Type type = SUPPORTED_TYPES.get(i);
-      if (type == Types.TimestampType.withZone() && isVectorized) {
-        // ORC/TIMESTAMP_INSTANT is not a supported vectorized type for Hive
+      if (isUnsupportedOrcVectorizedTypeForHive(type)) {
```

Comment:
I think this is a bit awkward. Wouldn't be better if we just create a single method, like:
`private static boolean vectorizationSupported(FileFormat fileFormat, Type type)`?

Reply:
Ok